src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/ProtoTest.kt

```diff
@@ -0,0 +1,13 @@
+package com.bardsoftware.papeeria.backend.cosmas
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+class ProtoTest {

```

Comment:
Add a short doc-comment and `@author` tag with your real name and github account:
```
/**
* This class <insert here a brief description of the purpose of this class>
* @author Alexander Fedotov (iisuslik43)
*/
```